src/go/k8s/config/e2e-test/manager.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        image: gcr.io/vectorized/redpanda-k8s-operator:latest
```

Comment:
let's rename the image here https://github.com/vectorizedio/redpanda/blob/87c08b9348f2e8a23a08dad0c13eff43712763da/src/go/k8s/Makefile#L3 and we won't need this?

Reply:
Ok I can do this.